modules/generator/processor/servicegraphs/servicegraphs.go

```diff
@@ -86,8 +86,20 @@ type Processor struct {
 
 func New(cfg Config, tenant string, registry registry.Registry, logger log.Logger) gen.Processor {
 	labels := []string{"client", "server", "connection_type"}
+
+	if cfg.EnableVirtualNodeLabel {
+		cfg.Dimensions = append(cfg.Dimensions, "virtual_node")
```

Comment:
should we make this slightly harder to conflict? double prefixed underscore (`__virtual_node`) seems like a common prometheus practice.

should we make this value a const?

Reply:
sounds good, I'm adding this